### ISSUE TITLE
Add support for AlarmRemoteController (rtds:RTDSRemoteControllerComponent)

### ISSUE THREAD

**Issue by clemalex824** [iMicknl/ha-tahoma#259]
**Device information**
https://boutique.somfy.fr/telecommande-tahoma-serenity.html
Remote control for TaHoma serenity, Somfy's connected security:
• 5-button remote control
• Manage the modes of TaHoma Serenity
• SOS function to generate alerts in case of emergency

**Device details**

2020-09-14 11:06:15 DEBUG (MainThread) [custom_components.tahoma] Unsupported TaHoma device detected (rtds:RTDSRemoteControllerComponent - RemoteController - AlarmRemoteController)

```json
{
	"commands": [],
	"states": [{
		"type": "ContinuousState",
		"qualifiedName": "rtds:ControllerBatteryState"
	}, {
		"type": "ContinuousState",
		"qualifiedName": "rtds:ControllerBipState"
	}, {
		"type": "ContinuousState",
		"qualifiedName": "rtds:ControllerOrderTypeState"
	}, {
		"type": "ContinuousState",
		"qualifiedName": "rtds:ControllerOriginatorState"
	}, {
		"type": "ContinuousState",
		"qualifiedName": "rtds:ControllerSensingState"
	}, {
		"type": "ContinuousState",
		"qualifiedName": "rtds:ControllerSirenState"
	}],
	"dataProperties": [],
	"widgetName": "AlarmRemoteController",
	"uiClass": "RemoteController",
	"qualifiedName": "rtds:RTDSRemoteControllerComponent",
	"type": "REMOTE_CONTROLLER"
}
```

**Comment by iMicknl** [comment]
What would be the use case for this device in Home Assistant? It looks like it is only possible to read the status, thus mapping it to a sensor is the only option.

Another option is to ignore this device in Home Assistant, which means that it won't show as an unsupported device in the log anymore?

**Comment by tetienne** [comment]
Can you see this device in your Tahoma interface?

**Comment by clemalex824** [comment]
No, i don't see the device in thaoma. I have activate the debug mode to view this.

I would like to be able to use it to open and close the gate which is not Somfy brand. (run script)

I bought 433Mhz modules and I would like, when pressing one of the buttons, to open or close the gate in full or pedestrian opening mode.

I have see that the button clicks activate / deactivate the alarm created by the integration and i can use this in automation.

But, the update delay is important and when I set the update interval to 3 seconds, I can no longer connect to the tahomalink interface because message of too many requests

**Comment by iMicknl** [comment]
Just to be clear, does it show up in the interface on tahomalink.com? I am not sure if this is an use case that we could easily support. Why not use another remote that you could use with Home Assistant? 

By the way;
- currently we don't support opening the gate in pedestrian opening mode, this could be added as a service in HA
- would you need a lower polling interval than 3 seconds? It should work with low intervals as well, are you using other integrations / programs with the same Somfy TaHoma account?

**Comment by clemalex824** [comment]
I use this remote control because it is  a device of the simple tahoma pack.

I will use a contactor for the different modes for the gate. I don't need any support for this..

The remote control appears in tahomalink.

Since this morning I have set an interval of 1 second which is sufficient and the message of too many requests is random...

As it is an accessory of the simple pack (I bought the tahoma in October 2019), I thought that the integration would be simple ...

I don't really need it because I can actually buy a 433mhz remote control but I just wanted to be able to use this one because otherwise it is in a drawer and is useless.

But when i use the Total Alarm Mode button, the state of alarm_control_panel.alarm of tahoma integration change.

so if you can add this remote that would be great (with all button support) but otherwise i will start using it by scanning the alarm.

However: 

Thanks for this component 👍

**Comment by clemalex824** [comment]
Should I close? Is a job going to be started?

**Comment by iMicknl** [comment]
> Should I close? Is a job going to be started?

For now, this hasn't been planned yet. There are some other issues which we prioritised at the moment. 
I have added the 'help wanted' tag, which means that the issue won't be closed automatically and that it is visible for people who would like to contribute.

**Comment by tetienne** [comment]
Let's imagine we finally successfully map this remote to a sensor. As we poll the Tahomalink website, it will me that once one button pressed you will have a delay with the action.

By the way how do you configure the buttons?

**Comment by clemalex824** [comment]
You are right to talk about the polling time because I set it to one second to have the least possible latency. If Somfy alerts when the time is too short, this remote control will be of no use. Thanks for the support but don't waste time on it (for my part).

**Comment by LoicRivoal** [comment]
Thanks for the effort 👍.
I would like to use this remote controller to trigger opening and close of a Wifi garage opener integrated into Home Assistant.

**Comment by iMicknl** [comment]
@LoicRivoal due to #289, the lowest update frequency has been changed to 30 seconds. This means that it could take up to 30 seconds before your remote controller command will be received by Home Assistant.

Due to this limitation, I don't see a good use-case for this remote controller. We won't spend time to implement this feature, but I will leave your issue open in case someone would like to contribute.

**Comment by iMicknl** [comment]
Anyone still interested in this sensor in HA? @LoicRivoal / @clemalex824? We can now easier add binary_sensor or sensors.

The polling time will still be high, however this could be lowered if you have local support. (currently only possible by jailbreaking your box).

**Comment by clemalex824** [comment]
I'm always interested. But I don't want to touch the firmware or other manipulation on the tahoma... 😱
